src/aiortc/contrib/media.py

```diff
@@ -208,6 +238,7 @@ async def recv(self):
 
     def stop(self):
         super().stop()
+        print("It is stopping in PlayerStreamTrack, error happens before this!")
```

Comment:
Does this always imply an error? Do you want a print statement here?

Reply:
removed the print